site/src/components/DeploySettingsLayout/OptionsTable.tsx

```diff
@@ -56,6 +61,29 @@ const OptionsTable: FC<{
   )
 }
 
+// optionValue is a helper function to format the value of a specific deployment options
+function optionValue(option: DeploymentOption): string[] | string | unknown {
+  switch (option.name) {
+    case "Max Token Lifetime":
```

Comment:
What's up with this empty case?

Reply:
Don't case blocks fall through to each other? These two cases are handled the same.

Reply:
:D I had to google that - you're right. 

Down the road, I wonder if we should type `DeploymentOption.flag` as something other than `string` (like we should return an enum). Then we could have an object lookup on the FE, like:

````
const deploymentOptionFormats: { [key: FlagEnum]: (option: DeploymentOption) => string[] | string | unknown } = {
    flagName1: (option) => {
        // format and return
    },
    ....
}

// in the JSX
<OptionValue>{deploymentOptionFormats[option.flag](option)}</OptionValue>
````

This would give us some additional type safety. Not something to worry about in this PR.

Reply:
:+1: Totally makes sense. I didn't know the "typescript" way to do this.